pkg/sdn/plugin/controller.go

```diff
@@ -384,10 +384,12 @@ func (plugin *OsdnNode) updateEgressNetworkPolicyRules(vnid uint32) {
 	} else if vnid == 0 {
 		glog.Errorf("EgressNetworkPolicy in global network namespace is not allowed (%s); ignoring", policyNames(policies))
 	} else if len(namespaces) > 1 {
+		// Rationale: If multiple namespaces have their own egress network policy, it's hard to determine the order of policies to apply. Also it may be wrong to force a network policy on a namespace which only wants to join the network.
```

Comment:
Mention something about VNIDs or NetNamespaces there since that's the source of the trouble; EgressNetworkPolicies are defined per-Namespace, but are implemented per-NetNamespace.